### PR TITLE
Multiifoplotsegments

### PR DESCRIPTION
**Pull request by tapaimarton**
This PR modifies the pycbc_page_segplot for multi-ifo use and implements it into the workflow.

### CODE REVIEW

File: bin/hdfcoinc/pycbc_page_segplot [hunk 1]
```diff
@@ -78,7 +84,7 @@ smin, smax = numpy.inf, -numpy.inf
 h = .7
 
 #FIXME: set IFO list
-ifos = ['H1', 'L1']
+ifos = opts.ifos
```
Comment (tdent) [inline bin/hdfcoinc/pycbc_page_segplot]: remove the FIXME, since you have just fixed it!!

File: bin/hdfcoinc/pycbc_page_segplot [hunk 2]
```diff
@@ -39,6 +39,8 @@ parser.add_argument('--segment-names', type=str, nargs="+", required=False,
                         help='Names of segments in the segment definer table.')
 parser.add_argument('--output-file', type=str,
                         help='Path of the output HTML file.')
+parser.add_argument('--ifos', nargs='+', default=['H1', 'L1'],
+                        help='List of IFOs: default H1, L1.')
```
Comment (tdent) [inline bin/hdfcoinc/pycbc_page_segplot]: it looks like this must be entered a space-separated list, whereas the help suggests a comma-separated list. make this clear in help message as for page_segtable

File: bin/hdfcoinc/pycbc_page_segplot (hunk 1 above)
Comment (tapaimarton) [reply bin/hdfcoinc/pycbc_page_segplot]: Done

File: bin/hdfcoinc/pycbc_page_segplot (hunk 2 above)
Comment (tapaimarton) [reply bin/hdfcoinc/pycbc_page_segplot]: Done

**Pull request merged**